common/views/components/DropdownButton/DropdownButton.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import { CSSTransition } from 'react-transition-group';
+import { useState, useRef, useEffect, type Element } from 'react';
+import styled from 'styled-components';
+import { classNames, font } from '../../../utils/classnames';
+import getFocusableElementsIn from '../../../utils/get-focusable-elements-in';
+import Space from '../styled/Space';
+import Icon from '../Icon/Icon';
+
+const DropdownWrapper = styled.div.attrs({
+  className: classNames({
+    relative: true,
+  }),
+})``;
+
+const ButtonEl = styled(Space).attrs(props => ({
```

Comment:
Why the `El`?
What about `ElButton` 🇲🇽 🇪🇸 ?

Reply:
Good question… errrrm… not sure. Will update.